processor/tailsamplingprocessor/processor_test.go

```diff
@@ -429,22 +431,32 @@ func generateIdsAndBatches(numIds int) ([]pdata.TraceID, []pdata.Traces) {
 	spanID := 0
 	var tds []pdata.Traces
 	for i := 0; i < numIds; i++ {
-		traceIds[i] = tracetranslator.UInt64ToTraceID(1, uint64(i+1))
+		traceID := [16]byte{}
+		binary.BigEndian.PutUint64(traceID[:8], 1)
+		binary.BigEndian.PutUint64(traceID[8:], uint64(i+1))
+		traceIds[i] = pdata.NewTraceID(traceID)
 		// Send each span in a separate batch
 		for j := 0; j <= i; j++ {
 			td := simpleTraces()
 			span := td.ResourceSpans().At(0).InstrumentationLibrarySpans().At(0).Spans().At(0)
 			span.SetTraceID(traceIds[i])
 
 			spanID++
-			span.SetSpanID(tracetranslator.UInt64ToSpanID(uint64(spanID)))
+			span.SetSpanID(UInt64ToSpanID(uint64(spanID)))
 			tds = append(tds, td)
 		}
 	}
 
 	return traceIds, tds
 }
 
+// UInt64ToSpanID converts the uint64 representation of a SpanID to pdata.SpanID.
+func UInt64ToSpanID(id uint64) pdata.SpanID {
```

Comment:
No need to be public -> rename `uInt64ToSpanID `

Reply:
I just renamed them to be `uInt64ToSpanID `.